data-structures-stacks/get-2nd-from-top.js

```diff
@@ -0,0 +1,14 @@
+/* exported get2ndFromTop */
+
+function get2ndFromTop(stack) {
+  let last = '';
+  if (stack.peek() === undefined) {
+    return undefined;
+  } else {
+    last = stack.peek();
+    stack.pop();
+  }
```

Comment:
Since there is a `return` in the above `if` statement you don't need an `else` here